### PR TITLE
Pin Godep to version v53

### PR DESCRIPTION
**Pull request by eparis**
Godep version v54+ started adding lots of new entries in Godep. It also removed needed dependency files! This pins the version we verify to v53. I'll work with godep to get them working.

### CODE REVIEW

**Comment by lavalamp** [comment]
So your godep wants to add a bunch of stuff, and mine wants to delete a bunch. ??

**Comment by lavalamp** [comment]
I see. I'll try getting that version and see what it does.

**Comment by lavalamp** [comment]
So this way, it produces no diff. This seems at least worth checking in for reproducibility. LGTM, thanks.

File: hack/verify-godeps.sh [hunk 1]
```diff
@@ -56,8 +56,11 @@ trap cleanup EXIT
 # build the godep tool
 export GOPATH="${_tmpdir}"
 go get -u github.com/tools/godep 2>/dev/null
-go install github.com/tools/godep 2>/dev/null
 GODEP="${_tmpdir}/bin/godep"
+pushd "${GOPATH}/src/github.com/tools/godep" > /dev/null
+  git checkout v53
+  "${GODEP}" go install
```
Comment (lavalamp) [inline hack/verify-godeps.sh]: So this uses the previous version to install the new version? Is that a good idea?

Comment (eparis) [inline hack/verify-godeps.sh]: It uses the latest and greatest to build the old version. If they can't even make godep build itself, they have serious problems...   works today, and hopefully we can revert this entirely shortly-ish

**Comment by goltermann** [comment]
@eparis Want me to just move these changes into my PR, or for you to submit this and me to rebase and resubmit?  I'm happy to do either.

**Comment by goltermann** [comment]
Saw your comments in #21721, I'll pull this change in that PR.

**Comment by lavalamp** [comment]
Yeah, @goltermann you can either pull this and rebase on top of it, or just wait for it to merge and then do your change again.

**Comment by eparis** [comment]
@k8s-bot unit test this flake: #21451

**Comment by thockin** [comment]
holy crap, can we get away from godep as quickly as possible?

**Pull request merged**